models/src/main/scala/coop/rchain/models/Error.scala

```diff
@@ -1,4 +1,4 @@
-package coop.rchain.storage
+package coop.rchain.models
```

Comment:
Let's keep this in this in `storage`.

In order to make this work, let's change `Serialize` to:

```scala
trait Serialize[A] {

  def encode(a: A): Array[Byte]

  def decode(bytes: Array[Byte]): Either[Throwable, A]
}
```

Reply:
Fixed.